tests/DocBlock/TypeExpressionTest.php

```diff
@@ -714,17 +714,6 @@ private function checkInnerTypeExpressionsStartIndex(TypeExpression $typeExpress
         return $res;
     }
 
-    private function clearPcreRegexCache(): void
```

Comment:
should we say we are not compatible with 8.2.8 etc?

Reply:
This is only for testing for a specific PCRE bug that is present /wo PCRE JIT but not /w PCRE JIT. Once regex caching is fixed to honor PCRE JIT flag is fixed in php-src, the manual cache clearing is not needed.

In simple words, this is here now to assert the fixer behaves correctly under all PCRE configurations, so we are compatible with any PHP version now and after.